tracks/ruby/exercises/high-scores/mentoring.md

```diff
@@ -59,4 +60,5 @@ Concatenation mutates the string instance in-place, `String#+` creates new strin
 Ruby 2.3 will freeze string literals by default.
 
 ### Changelog
-- Version 2 changed method names "highest" -> "personal_best", "top" -> "personal_top"
+- Version 3/4 changed method names "personal_top" -> "personal_top_three" and removed the "report" method.
```

Comment:
I would make this 2 lines, even though version 3 was never released by itself. But I don't feel strongly about it if you'd rather not.

Reply:
My thinking was: separate lines suggest that there are 2 extra versions around, and I hoped that this implicated there's only 1 newer version.